pkg/apis/rook.io/v1alpha2/annotations.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2018 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
```

Comment:
Space between license and `package ...` so that the license doesn't become godoc for the package.

Reply:
@BlaineEXE As I copied this from another Rook file and fixed it in some, we would need to fix this in many other files.